packages/reqresp/test/mocks/logger.ts

```diff
@@ -0,0 +1,11 @@
+import sinon from "sinon";
+import {ILogger} from "@lodestar/utils";
+
+export const createStubbedLogger = (): ILogger => ({
```

Comment:
Why is this thing needed?

Reply:
To reuse the stub where-ever logger required.

Reply:
I have some PTSD with unnecessary usage of Sinon https://github.com/ChainSafe/lodestar/issues/1523 Please avoid sinon unless you actually want to assert calls, even then there always better patterns to achieve the same test guarantees

In this specific case you can do

```ts
export const loggerNoop = {
  info: () => {}
  ...
}
```

Reply:
Or no need to re-invent the wheel, and just re-use testLogger which has been incredibly helpful to me to debug tests

Reply:
Based on my experience I would disagree to your conclusion for the sinon. If used properly and with right patterns, it's always best to use stubs and spies in the unit tests. There is no harm in using sinon based stubs to replace dependencies, even if we don't need to assert any of it's behavior. It's better approach for test mocks.